tests/Directory.Packages.props

```diff
@@ -3,11 +3,11 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
-    <PackageVersion Include="FluentAssertions" Version="6.12.0" />
+    <PackageVersion Include="FluentAssertions" Version="6.12.1" />
```

Comment:
**suggestion**: You could set the version as `6.12.*` or `6.*` and use [lock file](https://devblogs.microsoft.com/nuget/enable-repeatable-package-restores-using-a-lock-file/) to control the precise version and its dependencies.

Reply:
I'll consider in the next update!